pkgs/flox/default.nix

```diff
@@ -22,10 +22,10 @@ let
   version =
     if (FLOX_VERSION != null) then
       FLOX_VERSION
-    else if !(self ? revCount || self ? shortRev) then
-      "${fileVersion}-dirty"
-    else if !(self ? revCount) then
+    else if (self ? shortRev) then
       "${fileVersion}-g${self.shortRev}"
+    else if (self ? dirtyShortRev) then
+      "${fileVersion}-g${self.dirtyShortRev}"
```

Comment:
Is there any way to verify that we put the correct version in released containers before this is merged? cc @tomberek @bryanhonof @garbas

Reply:
We already do that at the end of a `postBuild` a bit later down in this nix file.

Reply:
It verifies that the version parses but not that it's correct (e.g. with no suffixes) at the time of release. Is there any way to be sure of that?